lib/grpc/server/transcode.ex

```diff
@@ -70,6 +69,8 @@ defmodule GRPC.Server.Transcode do
     Map.get(response_body, key)
   end
 
+  def map_response_body(%{}, response_body), do: response_body
```

Comment:
This doesn't match on an empty map. It matches on any map. You need to use when map_size(arg) == 0

Reply:
Also, please add tests for this